composer.json

```diff
@@ -10,10 +10,10 @@
         "psr-4": {"Lullabot\\Drainpipe\\Tests\\Functional\\": "tests/src/functional/"}
     },
     "require": {
-        "php": "^8.1",
+        "php": "^8.2",
         "ext-json": "*",
         "composer-plugin-api": "^2.0",
-        "drush/drush": "^11|^12|^13",
+        "drush/drush": "13.x-dev",
```

Comment:
We definitely need Drainpipe to support 11, 12, and 13. 
We can't force all projects to dev version of 13. But I understand we needed to do that to get Tugboat's `drush sql-sanitize` to work because of https://www.drupal.org/node/3431286

Reply:
@deviantintegral @justafish knowing that we can't (nor do we want to) force everyone to PHP 8.2 and a dev release of Drush 13, what are our options here to unblock pull requests that are only failing on Tugboat builds with this sessions table error?

Reply:
I think we should reach out to Moshe and ask about rolling a Drush 12 point release. Since this breaks for drupal 10.3 (and not 11) I think getting that out is the best thing?

Reply:
@deviantintegral to unblock our recent PRs would it be okay to pin the Drupal version while we wait for Drush to create new releases? https://github.com/Lullabot/drainpipe/pull/608